### PR TITLE
fix(Footer): update readme

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#4346 

### Description

Add `dds--privacy-cp` to stable selector documentation for both React and web components.

### Changelog

**New**

- `dds--privacy-cp` stable selector added to docs

**Changed**

- fixed typos in docs

### CODE REVIEW

File: packages/web-components/src/components/footer/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -146,5 +146,6 @@ See [our README](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/t
 | `<dds-footer-logo>`      | `data-autoid="dds--footer-logo"`       | Component   |
 | `<dds-locale-button>`    | `data-autoid="dds--footer-locale-btn"` | Interactive |
 | `<dds-legal-nav>`        | `data-autoid="dds--legal-nav"`         | Component   |
+| `<dds-privacy-cp>`       | `data-autoid="dds--privacy-cp"`        | Component   |
```
Comment (asudoh) [inline packages/web-components/src/components/footer/__stories__/README.stories.mdx:149]: To reflect the actual tag name in use:

```suggestion
| `<dds-legal-nav-cookie-preferences-placeholder>` | `data-autoid="dds--privacy-cp"`        | Component   |
```

**Review by asudoh — COMMENTED** [review]
Just one thing - Thanks @kennylam!

File: packages/web-components/src/components/footer/__stories__/README.stories.mdx (hunk 1 above)
Comment (kennylam) [reply packages/web-components/src/components/footer/__stories__/README.stories.mdx]: @asudoh Oops yes, you are right. Thanks!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @kennylam for the update!

**Pull request merged**